site/content/concepts/command-and-control.md

```diff
@@ -74,31 +74,18 @@ When a device connected to the AMQP adapter wants to receive commands from the a
 * `control` for authenticated devices
 * `control/${tenant}/${device-id}` for unauthenticated devices
 
-Once the receiver link is opened, the AMQP adapter sends command messages to devices through the link. The `subject` property of the message contains the actual command to be executed on the device while the `request_id` application property contains the identifier used to correlate the command request with the response.
+Once the receiver link is opened, the AMQP adapter sends command messages to devices through the link. The `subject` property of the request message contains the actual command to be executed on the device.
 
-Devices can publish command response messages by opening an **anonymous** sender link (similar to how `telemetry` and `event` messages are published) using the following message address:
-
-* `control` (authenticated device)
-* `control/${tenant}/${device-id}` (unauthenticated device)
-
-The device should set the `request_id` application property and the `status` application property on the command response message sent over the sender link. The `status` application property should contain the outcome of executing the command on the device.
-
-**Command Request Message Properties**
-
-The following table shows the properties that are set on a command request message by the AMQP adapter.
-
-| Name            | Mandatory       | Location                 | Type        | Description |
-| :-------------- | :-------------: | :----------------------- | :---------- | :---------- |
-| *subject*       | yes             | *properties*             | *string*    | Contains the command name to be executed on a device. |
-| *request_id*    | yes             | *application-properties* | *string*    | The request identifier used to correlate the command request with the response. |
+If the command request is a *one-way command*, then the device need not publish a command response message. However, if the application expects a response, then devices should publish the response by opening an **anonymous** sender link (similar to how `telemetry` and `event` messages are published). The device should set the message address, status and correlation-id properties of the response accordingly. Consult the table below for a list of properties that a device must set on a command response message.
```

Comment:
IMHO we should explicitly mention that the device may re-use an (anonymous) link that it uses for sending telemetry and/or events for this purpose, i.e. there is no need that the device opens another link ...